src/screens/Home/QuickStart/QuickStartUIUtils.res

```diff
@@ -221,23 +221,50 @@ module SelectConnectorGrid = {
       Processors(ADYEN),
       Processors(CHECKOUT),
     ]->Array.filter(connector => {
-      !(typedConnectedConnectorList->Array.includes(connector))
+      let filteredValue = typedConnectedConnectorList->Array.find(item => {
+        QuickStartUtils.filterConnectorFromArray(item, connector)
+      })
+
+      switch filteredValue {
+      | Some(_value) => false
+      | None => true
+      }
+    })
+
+    let remainingConnectorList = connectorList->Array.filter(value => {
+      let popularConnectorListValue = popularConnectorList->Array.find(item => {
+        QuickStartUtils.filterConnectorFromArray(item, value)
+      })
```

Comment:
If single line is there then {} is not needed

Reply:
Not a single line function

Reply:
`    let existsInArray = (element, array) =>
      array->Array.some(e =>
        switch (e, element) {
        | (Processors(p1), Processors(p2)) => p1 == p2
        | (_, _) => false
        }
      )
    let popularConnectorList =
      [
        Processors(STRIPE),
        Processors(PAYPAL),
        Processors(ADYEN),
        Processors(CHECKOUT),
      ]->Array.filter(fruit => !existsInArray(fruit, typedConnectedConnectorList))`
      use some rather than using find